processes_notwindows.go

```diff
@@ -115,15 +116,15 @@ func processesFromProc() ([]ProcStat, error) {
 func readProcFile(filename string) ([]byte, error) {
 	data, err := ioutil.ReadFile(filename)
 	if err != nil {
+		// if file doesn't exists it means that process was closed after we got the directory listing
```

Comment:
I noticed this addresses: https://github.com/cloudradar-monitoring/cagent/issues/12
You can include things like `Fixes #12` or `Closes #12` in your commit message and then the issue is automatically referenced and closed when merging the PR.